paper.tex

```diff
@@ -427,16 +427,22 @@ \subsection{Classes}
 It can be mapped to the following EO code, where a class becomes a factory of objects:
 
 \begin{ffcode}
++alias org.eolang.cage
++alias org.eolang.txt.sprintf
++alias org.eolang.io.stdout
++alias org.eolang.hamcrest.assert-that
+
 [] > book
   [i] > new
     cage 0 > id
     seq > @
       id.write i
       []
         [] > path
-          QQ.io.sprintf > @
-            "/tmp/%s.txt"
-            id
+          stdout > @
```

Comment:
@ivan-egorov42 I think that we do not need this object here, since it was absent before

Reply:
I thought that `path` should print message to console. Otherwise what's the point in it?

Reply:
@ivan-egorov42 The point is to get the path. If you make such changes, fix also a Ruby code.

Reply:
@Graur OK, I understood.